src/sidebar/components/tag-editor.js

```diff
@@ -49,6 +57,13 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
 
   const ie11 = useMemo(() => isIE11(), []);
 
+  /**
+   * Retrieve the current trimmed text value of the tag <input>
+   */
+  const pendingTag = () => inputEl.current.value.trim();
+  const hasPendingTag = () => pendingTag() && pendingTag().length > 0;
+  const clearPendingTag = () => (inputEl.current.value = '');
```

Comment:
I can't remember, does changing the value of an `input` programatically trigger the `input` event? If not, do you need to call `onTagInput` explicitly here?

Reply:
In (p)react, as I understand it, yes: the `onInput` and `onChange` events are "identical."